.github/workflows/release--build-hotfix.yml

```diff
@@ -1,24 +1,50 @@
 name: "[Deployment] Create a hotfix release"
 
-on:
-  workflow_dispatch:
-    inputs:
-      releaseNumber:
-        description: "Numéro de la release à hotfix (ex: 200)"
-        required: true
-        type: string
+run-name: Create a hotfix release from ${{ github.ref }}
 
-      releaseVersion:
-        description: "Nom du hotfix (ex: 200.0.2)"
-        required: true
-        type: string
+on: workflow_dispatch
 
 jobs:
+  check-worflow-ref:
+    name: Check worflow ref
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check worflow ref
+        run: |
+          if ! [[ ${{ github.ref }} == refs/heads/maint\/\/v* ]]; then
+              echo "This workflow can only be triggered from a maintenance branch (starting with a 'maint/v')"
+              exit 1
+          fi
+
+  find-hotfix-tag-number:
+    # Find the next hotfix version available
+    # Example: given branch maint/v200 and existing tags v200.0.0, v200.0.1 then output == 200.0.2
+    name: Find hotfix tag number
+    needs: check-worflow-ref
+    runs-on: ubuntu-latest
+    outputs:
+      hotfixTagNumber: ${{ steps.find-hotfix-tag-number.outputs.hotfixTagNumber }}
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.4
+        with:
+          persist-credentials: false
+          fetch-depth: 0 # Fetch all history to get all tags
```

Comment:
A-t-on besoin de checkout ?
ça ne m'a pas l'air utile pour lister les tag
sauf s'il n'y a même pas le projet sans cette action ?

Reply:
oui ça sert à lister les tags. 
sinon tu as une erreur "fatal: not a git repository (or any of the parent directories): .git" comme dans [ce run](https://github.com/pass-culture/pass-culture-main/actions/runs/3387012659/jobs/5627174156)